README.md

```diff
@@ -24,100 +24,56 @@ At [Viaduct](https://www.viaduct.ai/), we manage our Kubernetes resources via th
 
 
 ## Requirements
-- [Go](https://github.com/golang/go)
-- [kustomize](https://github.com/kubernetes-sigs/kustomize/) built with Go (See [details below](#kustomize-go-plugin-caveats))
+- [kustomize](https://github.com/kubernetes-sigs/kustomize/)
 - [SOPS](https://github.com/mozilla/sops)
 - gpg
 
 
 ## Installation Options
 
-### Go Plugin
-
-KSOPS was originally developed as a [kustomize Go plugin](https://kubernetes-sigs.github.io/kustomize/guides/plugins/#go-plugins). Up until *v2.2.0* this was the only installation option. To install, follow steps 0-3 of the [Getting Started section](#getting-started) and then run `make install`.
```

Comment:
Is it worth keeping this but under a history section or some such?

Reply:
@daurnimator What are your thoughts on a migration guide section to explain this + any upgrade instructions instead of a history section?

Reply:
sure? I'm not sure what thoughts you were hoping for :P